docs/preview/continuous-patching/README.md

```diff
@@ -0,0 +1,468 @@
+        scan task ID: dt21
+        patch status: ---Not Available---
+        patch date: ---Not Available---
+        patch task ID: ---Not Available---
+        last patched image: import:dotnetapp-manual-patched
+        workflow type: continuouspatchv1
+```
+
+If scan is a failure and NO previous patched image exists
+```sh
+image: import:dotnetapp-manual
+        scan status: Failed
+        scan date: 2024-09-13 21:05:58.841962+00:00
+        scan task ID: dt21
+        patch status: ---Not Available---
+        patch date: ---Not Available---
+        patch task ID: ---Not Available---
+        last patched image: ---Not Available---
+        workflow type: continuouspatchv1
+```
+
+If scan is currently running and a patched image exists
+```sh
+image: import:dotnetapp-manual
+        scan status: Running
+        scan date: 2024-09-13 21:05:58.841962+00:00
+        scan task ID: dt21
+        patch status: ---Not Available---
+        patch date: ---Not Available---
+        patch task ID: ---Not Available---
+        last patched image: import:dotnetapp-manual-patched
+        workflow type: continuouspatchv1
+```
+
+If scan is currently running and NO patched image exists
+```sh
+image: import:dotnetapp-manual
+        scan status: Running
+        scan date: 2024-09-13 21:05:58.841962+00:00
+        scan task ID: dt21
+        patch status: ---Not Available---
+        patch date: ---Not Available---
+        patch task ID: ---Not Available---
+        last patched image: ---Not Available---
+        workflow type: continuouspatchv1
+```
+
+If patch is currently running and a patched image exists
+```sh
+image: import:dotnetapp-manual
+        scan status: Succeeded
+        scan date: 2024-09-13 21:05:58.841962+00:00
+        scan task ID: dt21
+        patch status: Running
+        patch date: 2024-09-13 21:07:32.841962+00:00
+        patch task ID: xyz2
+        last patched image: import:dotnetapp-manual-patched
+        workflow type: continuouspatchv1
+```
+
+If patch is currently running and NO patched image exists
+```sh
+image: import:dotnetapp-manual
+        scan status: Succeeded
+        scan date: 2024-09-13 21:05:58.841962+00:00
+        scan task ID: dt21
+        patch status: Running
+        patch date: 2024-09-13 21:07:32.841962+00:00
+        patch task ID: xyz2
+        last patched image: ---Not Available---
+        workflow type: continuouspatchv1
+```
+
+## Canceling Running Tasks
+
+Sometimes you may need to cancel tasks which are currently running or waiting to run. For this purpose, please run the following CLI command:
+```sh
+az acr supply-chain workflow cancel-run -r <registryname> -g <resourcegroup> --type <continuouspatchv1>
+```
+
+This command will cancel all CSSC tasks within the registry with a status of “Running”, “Queued” and “Started”. The command will output a success or failure. Failure results will follow the failure pattern of the other workflow commands if the input is incorrect.
```

Comment:
CSSC is also mentioned here without explaining what it means. Please use another term like "Continuous Patching Tasks".